common/v2/services/EthService/utils/providerWrappers.ts

```diff
@@ -38,3 +40,10 @@ export const getTransactionReceiptFromHash = async (
     .catch(_ => {
       return undefined;
     });
+
+export const getTxStatus = (providerInstance: ProviderHandler, txHash: string) =>
+  getTransactionReceiptFromHash(txHash, providerInstance).then(({ status }: any) => {
+    // Status will return 0 or 1 which is used to indicate success or failure of tx
+    if (![0, 1].includes(status)) return;
```

Comment:
What does it mean when `status` is undefined or different from 0 and 1? 
Can we include it in the comment plz?